src/pages/messages/components/MessageList.tsx

```diff
@@ -36,7 +57,12 @@ export const TicketMessageList = ({ ticket }: MessageListProps) => {
             text={message.text}
             date={message.createdDate}
             username={message.sender}
+            canDeleteMessage={user && (canDeleteMessage || user.nvaUsername === message.sender)}
             backgroundColor={ticketColor[ticket.type]}
+            onDelete={() => deleteTicketMessageMutation.mutateAsync(message.identifier)}
+            isDeleting={deleteTicketMessageMutation.isPending}
+            confirmDialogTitle={t('my_page.messages.delete_message')}
+            confirmDialogContent={t('my_page.messages.delete_message_description')}
           />
```

Comment:
Ser det allerede er mange props her, og det vil skalere dårlig om det vil komme 5 nye props om/når det i fremtiden kommer ønske at man også skal kunne endre egen melding om kurator ikke enda har svar elns. Tenker bare høyt nå: har du vurdert noe sånt som å sende inn en komponent som håndterer dette selv?

Den komponenten kan selv sjekke om bruker har lov til å slette, hvordan sletting foregår, etc

Veldig grovt dette, men noe ala:
```suggestion
            messageMenu={<MessageItemMenu message={message} user={user} />}
          />
```

Jeg tenker noe ala `sortingComponent` i `ListPagination`. Som sagt veldig grovt dette, så vi kan ta en diskusjon på det om det ikke gir mening (som det kanskje ikke gjør heller 😅)